src/plugins/intellisense/view.tsx

```diff
@@ -0,0 +1,486 @@
+        }
+        renderables.push({
+          type: "param",
+          latex: t.str,
+          renderables: paramBody,
+        });
+      }
+    }
+  }
+
+  return renderables;
+}
+
+type DocStringRenderableNoParam =
+  | {
+      str: string;
+      type: "text";
+    }
+  | {
+      latex: string;
+      type: "math";
+    };
+
+type DocStringRenderable =
+  | {
+      type: "param";
+      latex: string;
+      renderables: DocStringRenderableNoParam[];
+    }
+  | DocStringRenderableNoParam;
+
+interface DocStringToken {
+  str: string;
+  type: "text" | "math" | "param";
+}
+
+let counter = 0;
+
+export class FormattedDocstring extends Component<{
+  docstring: () => DocStringRenderable[];
+  selectedParam: () => string;
+}> {
+  template() {
+    return (
+      <For
+        each={() => this.props.docstring().map((e, i) => [e, i] as const)}
+        key={() => counter++}
+      >
+        <div style={{ display: "inline" }} class="dsm-intellisense-docstring">
+          {([r, _]: [DocStringRenderable, number]) => (
+            <Switch key={() => r}>
+              {() => {
+                switch (r.type) {
+                  case "param": {
+                    const ltx = () => textModeExprToLatex(r.latex) ?? r.latex;
+
+                    if (
+                      addBracketsToIdent(this.props.selectedParam()) !== ltx()
+                    ) {
+                      return <span></span>;
+                    }
+                    return (
+                      <div class="pfc-doc-param">
+                        <DStaticMathquillView
+                          latex={ltx}
+                          config={{}}
+                        ></DStaticMathquillView>{" "}
+                        -
+                        <FormattedDocstring
+                          selectedParam={this.props.selectedParam}
+                          docstring={() => r.renderables}
+                        ></FormattedDocstring>
+                      </div>
+                    );
+                  }
+                  case "text":
+                    return <span>{() => r.str}</span>;
+                  case "math":
+                    return (
+                      <DStaticMathquillView
+                        latex={() => textModeExprToLatex(r.latex) ?? r.latex}
+                        config={{}}
+                      ></DStaticMathquillView>
+                    );
+                }
+              }}
+            </Switch>
+          )}
+        </div>
+      </For>
+    );
+  }
+}
+
+export class PartialFunctionCallView extends Component<{
+  partialFunctionCall: () => PartialFunctionCall | undefined;
+  partialFunctionCallIdent: () => BoundIdentifierFunction | undefined;
+  partialFunctionCallDoc: () => string | undefined;
+}> {
+  init() {}
+
+  template() {
+    return (
+      <div
+        style={() => ({
+          display: this.props.partialFunctionCall() ? undefined : "none",
+        })}
```

Comment:
This could be an `<If predicate={() => this.props.partialFunctionCall()}>`.

... or at least I think so. You scared me with using an `<If>` just a few lines down. Makes me think this `display:` is doing something sneaky.